### PR TITLE
Add FAQ entry on FS exceptions.

### LINKED ISSUE #179: Create a FAQ entry about files being read locally with FS enabled

### PR DESCRIPTION
**Pull request by t4lz**
close #179

### CODE REVIEW

File: content/docs/overview/faq.md [hunk 1]
```diff
@@ -108,6 +108,41 @@ More details can be found in this [GitHub discussion.](https://github.com/metalb
 
 ## Common Issues
 
+### A local file is read instead of a remote one, or my application cannot find a file that is only present in the cluster
```
Comment (eyalb181) [inline content/docs/overview/faq.md:111]:
```suggestion
### My application is trying to read a file locally even though `fs` is set to `remote` (or vice versa)
```

File: content/docs/overview/faq.md [hunk 2]
```diff
@@ -108,6 +108,41 @@ More details can be found in this [GitHub discussion.](https://github.com/metalb
...
+
+There are pre-defined exceptions to the set FS mode. That means that some files are read locally by default
```
Comment (eyalb181) [inline content/docs/overview/faq.md:113]: This is way too wordy in my opinion. I would just put:
mirrord has a list of paths that are read locally by default, and another list that are read remotely by default, regardless of the configured `fs` mode. You can override this behavior in the configuration. 

Here you can find all the pre-defined exceptions...

In order to override...

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/overview/faq.md (hunk 2 above)
Comment (t4lz) [reply content/docs/overview/faq.md]: https://github.com/metalbear-co/mirrord.dev/pull/182/commits/f47c45e9ae6cb5d9e281c79d1765719dad7c490a

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/overview/faq.md [hunk 3]
```diff
@@ -134,6 +134,29 @@ This could happen for several reasons:
 1. The local process is listening on a different port than the remote target. You can either change the local process to listen on the same port as the remote target (don't worry about the port being used locally by other processes), or use the []`port_mapping`  configuration](/docs/overview/configuration/#feature-network-incoming-port_mapping) to map the remote port to a local port.
 2. You're running with `network.incoming.mode` set to `mirror` on a cluster with a service mesh like Istio or Linkerd, which isn't currently supported. In this case, you should use the `--steal` flag instead.
 
+### My application is trying to read a file locally even though `fs` is set to `remote`
```
Comment (t4lz) [inline content/docs/overview/faq.md:137]: Actually `"remote"` is not a real mode, and also there are two modes that should read in the cluster. Moreover, saying it that way might give users the wrong impression that they have to set the `fs` config to something in order for mirrord to read remotely.
```suggestion
### My application is trying to read a file locally instead of in the cluster even though I did not configure mirrord to do that
```
What do you think?

Comment (eyalb181) [reply content/docs/overview/faq.md:137]: in the cluster => from the cluster/remote pod
And I'd drop everything after "in the cluster", I think it's implied

Comment (t4lz) [reply content/docs/overview/faq.md:137]:
```suggestion
### My application is trying to read a file locally instead of from the cluster
```

**Review by eyalb181 — APPROVED** [review]
